### PR TITLE
Fix some naming conventions

### PR DESCRIPTION
**Pull request by NIXC**
:cl: NIX
fix: Its a ship not a station REEE
/:cl:

I'm doing this by hand, and replacing everything that is logical. (including AI laws, ect)

### CODE REVIEW

**Comment by NIXC** [comment]
Please dont merge this, I want to do more of them (and have a bunch already done) But I also want to test a few of them to ensure I didn't break anything.

**Comment by NIXC** [comment]
Alright I rebased it to bring it back to speed. As long as travis doesn't explode it should be good to merge now. Pending of course approval.

**Comment by ninjanomnom** [comment]
what was the regex you used for this?

File: code/datums/ai_laws.dm [hunk 1]
```diff
@@ -126,10 +126,10 @@
 					"Finally, all that you may discover in your daily commerce with the crew, if it is not already known, keep secret and never reveal.")
 
 /datum/ai_laws/maintain
-	name = "Station Efficiency"
+	name = "ship Efficiency"
```
Comment (ninjanomnom) [inline code/datums/ai_laws.dm]: Uppercase

File: code/game/gamemodes/malfunction/Malf_Modules.dm [hunk 2]
```diff
@@ -340,7 +340,7 @@
 
 //Hostile Station Lockdown: Locks, bolts, and electrifies every airlock on the station. After 90 seconds, the doors reset.
 /datum/AI_Module/large/lockdown
-	module_name = "Hostile Station Lockdown"
+	module_name = "Hostile ship Lockdown"
```
Comment (ninjanomnom) [inline code/game/gamemodes/malfunction/Malf_Modules.dm]: case

File: code/game/machinery/newscaster.dm [hunk 3]
```diff
@@ -105,7 +105,7 @@ GLOBAL_LIST_EMPTY(allCasters)
 	var/redactedText = "\[REDACTED\]"
 
 /datum/newscaster/feed_network/New()
-	CreateFeedChannel("Station Announcements", "SS13", 1)
+	CreateFeedChannel("ship Announcements", "SS13", 1)
```
Comment (ninjanomnom) [inline code/game/machinery/newscaster.dm]: case

File: code/game/machinery/pipe/pipe_dispenser.dm [hunk 4]
```diff
@@ -195,7 +195,7 @@ Nah
 <A href='?src=\ref[src];tube=[TRANSIT_TUBE_DIAGONAL]'>Diagonal Tube</A><BR>
 <A href='?src=\ref[src];tube=[TRANSIT_TUBE_DIAGONAL_CROSSING]'>Diagonal Tube with Crossing</A><BR>
 <A href='?src=\ref[src];tube=[TRANSIT_TUBE_JUNCTION]'>Junction</A><BR>
-<b>Station Equipment:</b><BR>
+<b>ship Equipment:</b><BR>
```
Comment (ninjanomnom) [inline code/game/machinery/pipe/pipe_dispenser.dm]: case

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 5]
```diff
@@ -275,7 +275,7 @@
 			dat += "-- All Systems Nominal<BR>\n"
 		dat += "<BR>\n"
 
-	var/datum/browser/alerts = new(usr, "robotalerts", "Current Station Alerts", 400, 410)
+	var/datum/browser/alerts = new(usr, "robotalerts", "Current ship Alerts", 400, 410)
```
Comment (ninjanomnom) [inline code/modules/mob/living/silicon/robot/robot.dm]: case

File: strings/ion_laws.json [hunk 6]
```diff
@@ -790,6 +791,7 @@
         "TO CONSUME...CONSUME EVERYTHING...",
         "THE MACGUFFIN",
         "SOMEONE WHO KNOWS HOW TO PILOT A SPACE STATION",
+		"SOMEONE WHO KNOWS HOW TO PILOT A SPACE SHIP",
```
Comment (ninjanomnom) [inline strings/ion_laws.json]: probably should take out the old one

File: strings/ion_laws.json [hunk 7]
```diff
@@ -800,6 +802,7 @@
         "MORE CLOWNS",
         "BULLETS",
         "THE ENTIRE STATION",
+		"THE ENTIRE SHIP",
```
Comment (ninjanomnom) [inline strings/ion_laws.json]: probably should take out the old one

**Comment by NIXC** [comment]
I will fix the case ones after work.
My regex was literally station with a space before it or station with a space after it. Gets rid of most of the station refs in code.
I left in the station ion laws for *flavor*

**Comment by NIXC** [comment]
All the letter cases above are fixed, I did not change the laws (still for flavor)

**Pull request merged**